client/register/register.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+<h3>Cadastro</h3>
+<form *ngIf="active" #f="ngForm" novalidate (ngSubmit)="onSubmit()">
+  <div class="form-group row {{name.pristine?'':(name.valid?'has-success':'has-danger')}}">
```

Comment:
Bootstrap usa classes `.has-success`, `.has-warning` e `.has-danger` para estilizar o campo do formulário. O campo só terá a classe danger se houver erro no campo, exceto no início (campo vazio, ou `pristine`).